File: C++ Programs/Home Programs/count_primes.py
import math
count = 0
sq=0
prime = True
for  i in range(1, 1000001,2):
    sq = int(math.sqrt(i))
    prime = True
    for factor in range(3,sq+1,2):
        if i % factor == 0:
            prime = False
            break
    if prime:
        count+=1

print("No of primes are - ",str(count));

File: C++ Programs/Home Programs/count_primes.java
import java.lang.*;
public class count_primes {
    public static void main(String[]args) {
        int count = 0;
        double sq;
        boolean prime = true;
        for (int i = 1; i <= 10000000; i += 2)
        {
            sq = Math.sqrt(i);
            prime = true;
            for (int factor = 3; factor <= sq; factor += 2)
            {
                if (i % factor == 0)
                {
                    prime = false;
                    break;
                }
            }
            if (prime)
                count++;
        }
        System.out.println("No of primes are - " + count);

    }
}